metafix/src/main/java/org/metafacture/metafix/FixPath.java

```diff
@@ -276,7 +279,15 @@ else if (isReference(field)) {
         }
         else {
             if (!hash.containsField(field)) {
-                hash.put(field, Value.newHash().withPathSet(newValue.getPath()));
+                final Value value = Arrays.asList(ReservedField.$prepend.name(), ReservedField.$append.name())
```

Comment:
This list could be made `static` and unmodifiable; also, doesn't need to be a list (`Set.of(...)`).

Reply:
Replaced the collection with `||` as part of refactoring to avoid additional `tail()` call above.